packages/transport/src/modules/sync/SynchronizedCollection.ts

```diff
@@ -88,16 +88,21 @@ export class SynchronizedCollection implements IDatabaseCollection {
         const newObjectJson = newObject.toJSON();
 
         if (typeof globalThis.process === "object" && globalThis.process.env.CI) {
-            const oldDocUpdated = Serializable.fromUnknown(await this.parent.read(newObject.id.toString()));
+            const databaseObject = Serializable.fromUnknown(await this.parent.read(newObject.id.toString()));
 
-            const readDiff = jsonpatch.compare(oldDocUpdated.toJSON(), oldObject.toJSON());
+            const readDiff = jsonpatch.compare(databaseObject.toJSON(), oldObject.toJSON());
             if (readDiff.length > 0) {
                 // eslint-disable-next-line no-console
                 console.error(`
 The data that is currently updated got modified between it initial reading and this update.
 This will lead to an data loss and inconsistency.
 Here is the diff of the data:
 ${JSON.stringify(readDiff, null, 2)}
+Old Object from Database:
+${JSON.stringify(oldDoc, null, 2)}
+Object in Database:
```

Comment:
`Object in Database` is a very strange wording. Maybe `Object to be stored to the Database` is more accurate?

Reply:
or `Object to write`

Reply:
This is the Object from the DB that got written in the meantime since the last read, the `Object to write` would the `newObject`

Reply:
ehm, what? :D and what's oldDoc?

Reply:
The `oldDoc` is the document that was initial read by the calling functions.
And the `databaseObject` is the current document in the database.